pre.js

```diff
@@ -22,8 +22,6 @@ function pngquant(file, options, printF) {
 
 	/* Create command line options to passed using input `options` object */
 	for (var key in options) {
-		if (!options.hasOwnProperty(key)) continue;
```

Comment:
I know about 'for in' reading iterable keys in prototype, but this won't happen here, it's a plain object given by user